tests/test_jules_gh_feedback.py

```
@@ -69,334 +34,37 @@ def test_logs_with_triple_backticks_escaping(self):
             ],
         }
 
-        with tempfile.NamedTemporaryFile("w", delete=False, suffix=".json") as f:
+        telemetry_path = "/tmp/test_jules_telemetry.json"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_jules_telemetry.json"

(S108)